### PR TITLE
Fix for the VieraST60.conf

### PR DESCRIPTION
**Pull request by Sami32**
From the log of this user that have a ST60 not correctly recognized:
http://www.universalmediaserver.com/forum/download/file.php?id=3743

### CODE REVIEW

File: src/main/external-resources/renderers/Panasonic-VieraST60.conf [hunk 1]
```diff
@@ -12,11 +12,18 @@ RendererIcon = panasonic.png
 # User-Agent: Panasonic MIL DLNA CP UPnP/1.0 DLNADOC/1.50
 # X-PANASONIC-Registration: VklFUkEgU1Q2MCBTZXJpZXMAAAAAAAAAAAA=
 # friendlyName=VIERA ST60 Series
+#
+# # X-PANASONIC-DMP-Profile=""
+# X-PANASONIC-Registration: VklFUkEgU1Q2MAAAAAAAAAAAAAAAAAAAAAA=
```
Comment (Nadahar) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: Isn't this an unique code per unit and irrelevant in this context?

File: src/main/external-resources/renderers/Panasonic-VieraST60.conf [hunk 2]
```diff
@@ -12,11 +12,18 @@ RendererIcon = panasonic.png
 # User-Agent: Panasonic MIL DLNA CP UPnP/1.0 DLNADOC/1.50
 # X-PANASONIC-Registration: VklFUkEgU1Q2MCBTZXJpZXMAAAAAAAAAAAA=
 # friendlyName=VIERA ST60 Series
+#
+# # X-PANASONIC-DMP-Profile=""
```
Comment (Nadahar) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: Why the double comment symbol, and why include an empty profile?

File: src/main/external-resources/renderers/Panasonic-VieraST60.conf (hunk 1 above)
Comment (Sami32) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: It's not a unique code.
Every informations sended, as `X-PANASONIC-DMP-Profile=""`, could be pertinent for later differentiation use, so writing it in the renderer is safe. Not every one can remember everything and running after log.

File: src/main/external-resources/renderers/Panasonic-VieraST60.conf (hunk 2 above)
Comment (Sami32) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: Copy paste typo...

File: src/main/external-resources/renderers/Panasonic-VieraST60.conf (hunk 1 above)
Comment (Nadahar) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: @Sami32 Sorry, I don't understand what you mean by "later differentiation use". I never thought it was unsafe, I simply wondered if it was useful.

Comment (Sami32) [inline src/main/external-resources/renderers/Panasonic-VieraST60.conf]: Yes, it can be useful.

**Comment by Sami32** [comment]
@SubJunk Thanks for your approval :)

**Pull request merged**